Packages/vcs/Lib/VTKPlots.py

```diff
@@ -434,13 +434,13 @@ def plot(self,data1,data2,template,gtype,gname,bg,*args,**kargs):
       returned.update(self.plot3D(data1,data2,tpl,gm,ren,**kargs))
     elif gtype in ["text"]:
       if tt.priority!=0:
-        if not (None,None,None) in self._renderers.keys():
-            ren = self.createRenderer()
-            self.renWin.AddRenderer(ren)
-            self.setLayer(ren,1)
-            self._renderers[(None,None,None)]=ren
-        else:
-            ren = self._renderers[(None,None,None)]
+        #if not (None,None,None) in self._renderers.keys():
+        ren = self.createRenderer()
+        self.renWin.AddRenderer(ren)
+        self.setLayer(ren,1)
```

Comment:
It looks like this will have no effect, as the same call is made a few lines later with tt.priority for the layer.